src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Ticketing_TicketTabNavStack/TicketTabNav_PurchaseTabScreen/Components/FareProducts/FareProductTile.tsx

```diff
@@ -104,7 +157,85 @@ const useStyles = StyleSheet.createThemeHook((theme) => ({
   },
   title: {
     marginBottom: theme.spacings.small,
-    marginTop: theme.spacings.medium,
   },
   description: {marginBottom: theme.spacings.small},
+  coloredBottomLine: {
+    position: 'absolute',
```

Comment:
I'm not too familiar with React Native styling, but just seeing `position: absolute` gives me the creeps. Absolute positioning is usually something you only want to use in really special cases. Are you sure this is not possible through the use of `border` or similar?

Reply:
Also possible with border of course. To keep the same height as before, is there a way in RN to draw the border on the inside? If not we can just reduce borderBottomWidth.